src/cider/nrepl/middleware/complete.clj

```diff
@@ -2,33 +2,46 @@
   (:require
    [cider.nrepl.middleware.util.cljs :as cljs]
    [cider.nrepl.middleware.util.error-handling :refer [with-safe-transport]]
-   [cljs-tooling.complete :as cljs-complete]
-   [compliment.core :as jvm-complete]
-   [compliment.utils :as jvm-complete-utils]
+   [compliment.core :as complete]
+   [compliment.utils :as complete-utils]
    [orchard.misc :as misc]
-   [suitable.complete-for-nrepl :as suitable]))
+   [suitable.complete-for-nrepl :as suitable]
+   [suitable.compliment.sources.cljs :as suitable-sources]))
 
-(defn- cljs-complete
-  [{:keys [enhanced-cljs-completion?] :as msg} cljs-env ns prefix extra-metadata]
-  (concat (cljs-complete/completions cljs-env prefix {:context-ns ns
-                                                      :extra-metadata extra-metadata})
-          (when enhanced-cljs-completion? (suitable/complete-for-nrepl msg))))
+(def +clj-sources+
+  "Source keywords for Clojure completions."
+  [:compliment.sources.special-forms/literals
+   :compliment.sources.class-members/static-members
+   :compliment.sources.ns-mappings/ns-mappings
+   :compliment.sources.resources/resources
+   :compliment.sources.keywords/keywords
+   :compliment.sources.local-bindings/local-bindings
+   :compliment.sources.class-members/members
+   :compliment.sources.namespaces-and-classes/namespaces-and-classes
+   :compliment.sources.special-forms/special-forms])
+
+(def +cljs-sources+
```

Comment:
Why the + signs?

Reply:
it is a (questionable :smile:) convention I got from boot for globals that are not `^:dynamic`. I can get rid of it if we don't like it.

Reply:
Seems weird to me, plus it's not consistent with the rest of our code. Let's remove them.

Reply:
Will do